src/transforms/log_to_metric.rs

```diff
@@ -184,18 +200,45 @@ impl LogToMetric {
     }
 }
 
+/// Kinds of TranformError for Parsing
+#[configurable_component]
+#[derive(Clone, Debug)]
+pub enum TransformParseErrorKind {
+    ///  Error when Parsing a Float
+    FloatError,
+    ///  Error when Parsing an Int
+    IntError,
+    /// Errors when Parsing Arrays
+    ArrayError,
+}
+
+impl fmt::Display for TransformParseErrorKind {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{:?}", self)
+    }
+}
+
 enum TransformError {
     FieldNotFound {
```

Comment:
Let's also rename `Field` to `Path` (more generic)

Reply:
Done.